earth_enterprise/src/third_party/skia/SConscript

```diff
@@ -100,6 +100,8 @@ sgl_env.UpdateCppflagsForSkia()
 
 cxx_flags = sgl_env['CXXFLAGS']
 cxx_flags.remove('-Werror')
+cxx_flags.append('-fPIC')
```

Comment:
I'm sorry, I don't recall why I had to add that. It was several months ago. It has to do with how .so libraries are built on certain platforms. If someone wants to test whether it's still needed on CentOS 6, that would be great. But, I don't really have time right now. I can add a genericish comment with a link to the documentation.